state/stateConsensus.go

```diff
@@ -728,8 +727,6 @@ processholdinglist:
 		switch validToExecute {
 		case -1:
 			s.LogMessage("executeMsg", "invalid from holding", v)
-			TotalHoldingQueueOutputs.Inc()
```

Comment:
Do we care about losing all these metrics?

Reply:
Clay let me know that there was a bug here - removing these counters is desired.